omnigibson/objects/controllable_object.py

```diff
@@ -396,11 +395,13 @@ def deploy_control(self, control, control_type, indices=None, normalized=False):
 
         # Standardize normalized input
         n_indices = len(indices)
-        normalized = normalized if isinstance(normalized, Iterable) else [normalized] * n_indices
 
         # Loop through controls and deploy
         # We have to use delicate logic to account for the edge cases where a single joint may contain > 1 DOF
         # (e.g.: spherical joint)
+        pos_vec, pos_idxs, using_pos = [], [], False
+        vel_vec, vel_idxs, using_vel = [], [], False
+        eff_vec, eff_idxs, using_eff = [], [], False
```

Comment:
These change is needed for osc controllers to work (c.f. @cremebrule) because if we set the joint target sequentially the previous changes will be overwritten